user-mgt/custom-jdbc-user-store-manager/pom.xml

```diff
@@ -117,10 +123,11 @@
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Bundle-Name>${project.artifactId}</Bundle-Name>
                         <Private-Package>
-                            org.wso2.custom.user.store.internal.*
+                            org.wso2.custom.user.store.internal.*,
+                            org.jasypt.*

```

Comment:
Why this package is added as a private package.

Reply:
The sample application uses the jasypt library to hash the password. So in order to run this, the jasypt library should be available in the runtime. That's why I'm private packaging it.